test/integration/imagesigning_test.go

```diff
@@ -0,0 +1,130 @@
+// +build integration
+
+package integration
+
+import (
+	"bytes"
+	"reflect"
+	"testing"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/util/diff"
+
+	"github.com/openshift/origin/pkg/cmd/admin/policy"
+	"github.com/openshift/origin/pkg/cmd/server/bootstrappolicy"
+	imageapi "github.com/openshift/origin/pkg/image/api"
+	testutil "github.com/openshift/origin/test/util"
+	testserver "github.com/openshift/origin/test/util/server"
+)
+
+func TestImageAddAndRemoveSignature(t *testing.T) {
```

Comment:
I'd like to see coverage of:
 1. delete signature that doesn't exist
 1. delete correct signature of at least two signatures
 1. create signature conflicting by content, but not name